docs/id/docs/tutorial/first-steps.md

```diff
@@ -0,0 +1,334 @@
+# First Steps
```

Comment:
Great work @bas-baskara, shouldn't the index and title also need to be translated? [Here an example](https://github.com/tiangolo/fastapi/pull/1259/files#diff-36f5eba55d0de6739fbf682714cf8346d50ce364387653b6abdff54efce927ddR28) how they translate the index

Reply:
Hi @negadive

Based on the guidelines from https://fastapi.tiangolo.com/contributing/#existing-language: 

"Add a single pull request per page translated. That will make it much easier for others to review it."

So I translated this page for one PR. I've made another Pull Request to translate the index page.